app/assets/javascripts/app/project/project_sidebar.js

```diff
@@ -7,7 +7,8 @@ var baseSidebar = {
 
   selectReward: function(event){
     var url = this.$(event.currentTarget).data('new_contribution_url');
-    this.navigate(url);
+    var acceptContributions = this.$(event.currentTarget).data('accept_contributions');
```

Comment:
@igorrfc even it being ok to have a `_` inside a data attribute name it is not recommended since it's not a common pattern. Maybe the other parts of the project could be refactored to only use `-` as a separator.